README.md

```diff
@@ -1,3 +1,77 @@
 # @comet/dev-process-manager
 
+## Installation
+
+```console
+$ npm install
+$ npm run build
+```
+
 ## Usage
+
+Add `pm.config.js` file to the project root.
```

Comment:
I'd vote vor dev-pm.config.js

Reply:
The name used does not matter
I changed it in the readme to dev-pm.config.js